packages/components/package.json

```diff
@@ -26,20 +26,11 @@
     "styles.css"
   ],
   "scripts": {
-    "prepublishOnly": "pnpm build:components && pnpm build:styles && pnpm postBuild && pnpm dist:clean",
-    "build": "pnpm clean && pnpm prepublishOnly",
+    "build": "pnpm package-bundler build-shared-ui && pnpm styles:global",
+    "styles:global": "postcss ./styles/global.css --output dist/styles.css",
```

Comment:
generally across tools we use a pattern for commands [like here](https://github.com/cultureamp/next-template/blob/main/apps/app-template/package.json#L10-L11) which in this case could be `build:styles` as previously or if you want to highlight it's just the global styles `build:styles:global` or `build:global-styles`